### ISSUE TITLE
Vehicle Location geojson

### ISSUE THREAD

**Issue by aaronsinclair** [vingerha/gtfs2#43]
**Before Describe the bug**
Thank you for your generosity and time in making this integration  and sharing it with the community. 

**Describe the bug**
Real time vehicle position does not appear to be called anywhere in the code base as is described in the readme.md

```Option to add gtfs realtime vehicle location source/url, generates geojson file which can be used for tracking vehicle on map card```

Looking at the code I can see methods for 
* get_rt_vehicle_positions
* update_geojson

but I am unable to find any reference of this being called anywhere in the code. 

Is this feature currently implemented? 

Thank you

**Comment by aaronsinclair** [comment]
I found in  your older code you were calling get_rt_vehicle_positions from within another helper, this appears to have been removed. 

I have added the following locally to test: 

def get_rt_alerts(self):
    
    if self._vehicle_position_url != "" :   
        get_rt_vehicle_positions(self)

.......................

This is now generating the expected ./www/gtfs2/{route}_0.json file with appropriate contents.

**Comment by aaronsinclair** [comment]
I also note documentation as follows

_Note: geojson currently only updates each 5 minutes and can not be refreshed by a service call_

**Comment by vingerha** [comment]
The last version had a major rework in it and code cleaning, although I did think about veh.loca ... apparently this did not make it to the release. Will add it by eow. 
On geojson, this is core functinoality used for reading the geojson file(s) that gtfs2 generates and is limited to 5 mins. which I cannot change (dev. also does not respond to my request). In the meantime I learned how to create/remove entities at a higher frequency so a future implementation will allow faster updates
